src/portals/blockchain-frontend/apps/blockchain-frontend/components/login/state.ts

```diff
@@ -1,4 +1,5 @@
 import { useRouter } from 'next/router';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
```

Comment:
why this line?

Reply:
import type {
  UserRequest,
  UserResponse,
} from 'apps/blockchain-frontend/interfaces/viewModels';  

getting error in this import 

Projects cannot be imported by a relative or absolute path, and must begin with a npm scopeeslint[@nrwl/nx/enforce-module-boundaries](file:///)